app/models/planet.py

```diff
@@ -0,0 +1,18 @@
+from app import db
+
+class Planet(db.Model):
+        __tablename__ = 'planets'
+
+        id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+        name = db.Column(db.String, nullable=False)
```

Comment:
Great job making your planets non-nullable! We would definitely want to make sure we include a name when we create a planet!